Assets/MixedRealityToolkit-SDK/Models/Cube.prefab

```diff
@@ -0,0 +1,98 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
can we do something other than a cube?

Reply:
We could but MVP.
I'll see about using a different controller model in the devices PR.